packages/backend/src/tests/propertyBased/properties/executeQuery.ts

```diff
@@ -0,0 +1,34 @@
+import { Pool } from 'pg';
+import { QueryEngine, collectLast } from '../../..';
+import { AnySchema, getTableNamesFromGenericDbSchema } from '@synthql/queries';
+
+export async function executeQuery<DB>(
+    queryEngine: QueryEngine<DB>,
+    query: any,
+): Promise<any> {
+    const queryResult = await collectLast(
```

Comment:
Maybe we should move this function to the `QueryEngine` itself. It's very often that you _just_ need the last value.

What do you think could be a good name for it? (maybe `executeAndWait`)?